_tests/tests_suite_test.go

```diff
@@ -0,0 +1,98 @@
+package _tests_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/config"
+
+	. "github.com/onsi/gomega"
+
+	"testing"
```

Comment:
I agree, but apparently that's how [goimports](https://github.com/bradfitz/goimports) orders them. :sheep: